### PR TITLE
Add manual testing

### PR DESCRIPTION
**Pull request by tiongMax**
Added manual testing for eedit and edelete in dg.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -856,6 +856,48 @@ testers are expected to do more *exploratory* testing.
...
+       `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy m/100 packets b/100.00 vs/50`
+
+    1. Test case: `edelete 1`
+       Expected: The first event in the displayed event list is deleted successfully. Detail is shown in the status message.
+
+    1. Test case: `edelete`
+       Expected: No change to the event list panel. Invalid command format error is shown in the status message.<br>
+
+    1. Test case: `edele`
+       Expected: No change to the event list panel. Invalid command format error is shown in the status message.<br>
```
Comment (zacwong2151) [inline docs/DeveloperGuide.md:873]: when 2 test cases produces the same expected result, combine them into one

1. Other incorrect delete commands to try: `efind`, `efind randomwordshere n/baking cookies`, `...` <br>
       Expected: `Invalid command format!` error message shown in the status message. Instructions on the correct format are shown as well.
       
like this

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -856,6 +856,48 @@ testers are expected to do more *exploratory* testing.
...
+       create one with the following example, 
+       `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy m/100 packets b/100.00 vs/50`
+
+    1. Test case: `eedit 1 l/NUS`
+       Expected: The location of the first event in the displayed event list is edited from to NUS successfully. 
+       Detail is shown in the status message.
+
+    1. Test case: `eedit 1 sd/22/10/2023 1900`
+       Expected: The start date of the first event in the displayed event list is edited to 22/10/2023 1900 successfully.
+       Detail is shown in the status message.
```
Comment (zacwong2151) [inline docs/DeveloperGuide.md:887]: similar test cases, just combine both into one test case `eedit 1 l/NUS sd/22/10/2023 1900`

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -856,6 +856,48 @@ testers are expected to do more *exploratory* testing.
...
+
+    1. Test case: `eedit 1 r/`
+       Expected: The roles of the first event in the displayed event list is replaced to empty successfully.
+       Detail is shown in the status message.
+
+    1. Test case: `eedit 1 l/`
+       Expected: No change to the event list panel. Invalid command format error is shown in the status message.<br>
+
+    1. Test case: `eeit`
+       Expected: No change to the event list panel. Invalid command format error is shown in the status message.<br>
```
Comment (zacwong2151) [inline docs/DeveloperGuide.md:897]: similar test cases

**Review by zacwong2151 — CHANGES_REQUESTED** [review]
lil stuff

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (zacwong2151) [reply docs/DeveloperGuide.md:873]: or u can do it similar to what rx did and say `Expected: similar to previous`

Comment (tiongMax) [reply docs/DeveloperGuide.md:873]: oh sounds good

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (tiongMax) [reply docs/DeveloperGuide.md:887]: okeh

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -854,7 +854,52 @@ testers are expected to do more *exploratory* testing.
...
+       create it with the following example,
+       `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy m/100 packets b/100.00 vs/50`
+
+    1. Test case: `edelete 1`
+       Expected: The first event in the displayed event list is deleted successfully. Detail is shown in the status message.
+
+    1. Test case: `edelete`
+       Expected: No change to the event list panel. Invalid command format error is shown in the status message.<br>
+
+    1. Test case: `edele`
```
Comment (ruo-x) [inline docs/DeveloperGuide.md:872]: error message for `edele` is invalid command not invalid command format, also i think no need for this test case actl thoughts?

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -854,7 +854,52 @@ testers are expected to do more *exploratory* testing.
...
+       Detail is shown in the status message.
+
+    1. Test case: `eedit 1 r/`
+       Expected: The roles of the first event in the displayed event list is replaced to empty successfully.
+       Detail is shown in the status message.
+
+    1. Test case: `eedit 1 l/`
+       Expected: Similar to previous test case.<br>
+
+   1. Test case: `edelete`
```
Comment (ruo-x) [inline docs/DeveloperGuide.md:896]: y sudd got `edelete` here?

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -854,7 +854,52 @@ testers are expected to do more *exploratory* testing.
...
+      Expected: No change to the event list panel. Invalid command format error is shown in the status message.<br>
+   
+   1. Test case: `eeit`
```
Comment (ruo-x) [inline docs/DeveloperGuide.md:899]: still feel that invalid commands can dun add

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -854,7 +854,52 @@ testers are expected to do more *exploratory* testing.
...
+      Expected: Similar to previous test case.<br>
+
+1. _{ more test cases …​ }_
```
Comment (ruo-x) [inline docs/DeveloperGuide.md:902]: missed out on `eedit` , `eedit 0`, `eedit x` (where x is larger than size of list) test cases
AND editing the start date to be later than end date/vice versa test case!

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -854,7 +854,52 @@ testers are expected to do more *exploratory* testing.
...
     1. Other incorrect delete commands to try: `vdelete`, `vdelete x`, `...` (where x is larger than the list size or a non-integer)<br>
-       Expected: Similar to previous.
+       Expected: Similar to previous test case.
+
+### Deleting an event
+
+1. Deleting an event.
+    1. Prerequisites: List all events using the `elist` command. Multiple events in the list. If the list is empty,
+       create it with the following example,
+       `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy m/100 packets b/100.00 vs/50`
```
Comment (jordankanghm) [inline docs/DeveloperGuide.md:864]: y u copy my one :(

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (jordankanghm) [reply docs/DeveloperGuide.md:872]: me 2

File: docs/DeveloperGuide.md [hunk 9]
```diff
@@ -854,7 +854,52 @@ testers are expected to do more *exploratory* testing.
...
+
+    1. Test case: `edelete 1`
+       Expected: The first event in the displayed event list is deleted successfully. Detail is shown in the status message.
+
+    1. Test case: `edelete`
+       Expected: No change to the event list panel. Invalid command format error is shown in the status message.<br>
+
+    1. Test case: `edele`
+       Expected: Similar to previous test case.<br>
+
```
Comment (jordankanghm) [inline docs/DeveloperGuide.md:874]: u wanna add a test case for invalid index?

File: docs/DeveloperGuide.md [hunk 10]
```diff
@@ -854,7 +854,52 @@ testers are expected to do more *exploratory* testing.
...
+### Editing an event
+1. Editing the detail of an event.
```
Comment (jordankanghm) [inline docs/DeveloperGuide.md:876]: details!

File: docs/DeveloperGuide.md [hunk 11]
```diff
@@ -854,7 +854,52 @@ testers are expected to do more *exploratory* testing.
...
+    1. Prerequisites: List all events using the `elist` command. Multiple events in the list. If the list is empty, 
+       create one with the following example, 
+       `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy m/100 packets b/100.00 vs/50`
+
+    1. Test case: `eedit 1 l/NUS`
+       Expected: The location of the first event in the displayed event list is edited from to NUS successfully.
```
Comment (jordankanghm) [inline docs/DeveloperGuide.md:882]: i think u forgot to say what it is edited from

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (jordankanghm) [reply docs/DeveloperGuide.md:887]: on top of this i think u can add on a point at the end smth like "For all valid parameters and values, the field to be edited is updated successfully and shown in the status message" to generalise for all fields. but up to u tho if u find this useful

**Review by jordankanghm — CHANGES_REQUESTED** [review]
Just some minor changes to add on to what the others have already mentioned!

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (zacwong2151) [reply docs/DeveloperGuide.md:872]: me 3

File: docs/DeveloperGuide.md (hunk 5 above)
Comment (zacwong2151) [reply docs/DeveloperGuide.md:896]: exhibiting symptoms of the copy pasta

File: docs/DeveloperGuide.md (hunk 8 above)
Comment (brein62) [reply docs/DeveloperGuide.md:864]: change to `Food cooking` hehe

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (brein62) [reply docs/DeveloperGuide.md:872]: me 4

File: docs/DeveloperGuide.md [hunk 12]
```diff
@@ -854,7 +854,52 @@ testers are expected to do more *exploratory* testing.
...
+       Expected: No change to the event list panel. Invalid command format error is shown in the status message.<br>
+
+    1. Test case: `edele`
+       Expected: Similar to previous test case.<br>
+
+### Editing an event
+1. Editing the detail of an event.
+    1. Prerequisites: List all events using the `elist` command. Multiple events in the list. If the list is empty, 
+       create one with the following example, 
+       `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy m/100 packets b/100.00 vs/50`
```
Comment (brein62) [inline docs/DeveloperGuide.md:879]: interesting, good idea

File: docs/DeveloperGuide.md (hunk 11 above)
Comment (brein62) [reply docs/DeveloperGuide.md:882]: `hougang` to `NUS`

File: docs/DeveloperGuide.md (hunk 6 above)
Comment (brein62) [reply docs/DeveloperGuide.md:899]: same

**Review by brein62 — CHANGES_REQUESTED** [review]
like what everyone else has said haha

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (tiongMax) [reply docs/DeveloperGuide.md:872]: me 5!!!!

File: docs/DeveloperGuide.md (hunk 5 above)
Comment (tiongMax) [reply docs/DeveloperGuide.md:896]: HAHAHAHH zac shhhh dun expose me, it is just typo

File: docs/DeveloperGuide.md (hunk 6 above)
Comment (tiongMax) [reply docs/DeveloperGuide.md:899]: yessir!

File: docs/DeveloperGuide.md (hunk 8 above)
Comment (tiongMax) [reply docs/DeveloperGuide.md:864]: aiya got caught ady

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (ruo-x) [reply docs/DeveloperGuide.md]: yessirr

**Review by zacwong2151 — APPROVED** [review]
@tiongMax just fix merge conflicts and merge this, once done tell me, ill check everything again

**Review by brein62 — APPROVED** [review]
whatever zac said ^

**Comment by jordankanghm** [comment]
whatever brendan said^

**Comment by tiongMax** [comment]
Though i fixed everything ady

**Pull request merged**